components/slider/index.css

```diff
@@ -10,63 +10,94 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-slider.css";
+
+
+/* missing tokens for variant with Tick, also need t-shirt sizing */
+.spectrum--medium {
+  --spectrum-slider-tick-mark-height: 10px;
+  /* --spectrum-global-dimension-size-125 */
+  --spectrum-slider-tick-mark-width: 2px;
+  /* --spectrum-global-dimension-size-25: */
+  --spectrum-slider-tick-mark-border-radius: 2px;
+  /* --spectrum-global-dimension-size-25 2px for Spectrum,   --spectrum-global-dimension-size-10 1px for Express */
+  --spectrum-slider-ramp-track-height: 16px;
```

Comment:
`--spectrum-slider-ramp-track-height: 16px;`

I think this might need to just be a block size of 100%. After the t-shirt sizes are applied to the controls, it seems to work out right.

Reply:
<img width="520" alt="Screen Shot 2023-01-24 at 17 21 30" src="https://user-images.githubusercontent.com/52184321/214451678-fed9447a-aeb0-40f0-9579-dd1f39cc1933.png">

I don't think that works. The control height is a different size and the ramp height. 
We should still keep the var `--spectrum-slider-ramp-track-height` and replace it with tokens.